e2e/tests/slash_commands.spec.ts

```diff
@@ -33,9 +33,13 @@ test.describe('slash commands', () => {
         await expect(page.locator(`#sidebarItem_calls${userIdx}`).getByTestId('calls-sidebar-active-call-icon')).toBeHidden();
     });
 
-    test('end call as second host', async () => {
-        const user0Page = await startCall(userStorages[0]);
-        const user1Page = await joinCall(userStorages[1]);
+    test('end call as second host', async ({page}) => {
+        const user0Page = new PlaywrightDevPage(page);
+
+        const [, user1Page] = await Promise.all([
+            user0Page.startCall(),
+            joinCall(userStorages[1]),
+        ]);
```

Comment:
Here and in a few places that expect certain permissions on a given user, we are potentially introducing flakiness since the host is the first user to join and now both users join in parallel. That said, in one case (the second user) we have to spawn a new browser process, so I am assuming the first user will consistently beat that, guaranteeing the order. If that doesn't happen to be the case, I'll revert.

Reply:
Is there a reason you need to do this part in parallel then?

Reply:
@cpoile The reason is merely to speed up. Are you concerned?

Reply:
Maybe a comment then that this is a potential issue, just to remind someone else if it's not you. 👍

Reply:
Done